components/raftstore-v2/src/raft/peer.rs

```diff
@@ -641,6 +660,34 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             .advance_apply(apply_index, term, region);
     }
 
+    /// Register self to apply_scheduler so that the peer is then usable.
+    /// Also trigger `RegionChangeEvent::Create` here.
+    pub fn activate<T>(&mut self, ctx: &mut StoreContext<EK, ER, T>) {
```

Comment:
Why create a function for this, I see it's only called once.

Reply:
After the change of split init, yes, it is.